compiler/src/iree/compiler/Dialect/Stream/IR/test/invalid.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: iree-opt --split-input-file %s --verify-diagnostics
```

Comment:
these can be tested in executable_ops.mlir alongside the other ones, right?

Reply:
yes, I'm just used to `invalid.mlir` and didn't see one. Moved now.